docs/integrations/sources/mssql-migrations.md

```diff
@@ -1,5 +1,9 @@
 # Microsoft SQL Server (MSSQL) Migration Guide
 
+## Upgrading to 4.0.0
+Source MSSQL provides Incremental sync that can read unlimited sized table and can resume if initial read has failed.
```

Comment:
Source MSSQL provides incremental sync that can read unlimited sized tables and can resume if the initial read has failed.
Upgrading from previous versions will be seamless and does not require any intervention.